controllers/storageclassclaim/storageclassclaim_controller.go

```diff
@@ -20,65 +20,98 @@ import (
 	"context"
 	"fmt"
 
+	"github.com/go-logr/logr"
 	v1 "github.com/red-hat-storage/ocs-operator/api/v1"
+	"github.com/red-hat-storage/ocs-operator/api/v1alpha1"
 	ocsv1alpha1 "github.com/red-hat-storage/ocs-operator/api/v1alpha1"
 	"github.com/red-hat-storage/ocs-operator/controllers/storagecluster"
+	controllers "github.com/red-hat-storage/ocs-operator/controllers/storageconsumer"
+	rookCephv1 "github.com/rook/rook/pkg/apis/ceph.rook.io/v1"
 	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
k8s.io deps should be in different block than github.com deps. If you use goimports, it should be automatically formated.

Reply:
ack